game/global_objects.py

```diff
@@ -372,14 +372,15 @@ def check_ver_coll(self, ball):
                 did_collide = True
         return did_collide
 
-    def update(self, speed):
+    def update(self, speed, mute):
         if speed <= 3:
             return 0
         self.brick_value -= old_div(speed, MAX_BALL_SPEED)
         if self.brick_value <= 0:
             # add sound for breaking
-            break_sound.set_volume(2)
-            break_sound.play()
+            if mute==1:
```

Comment:
Shouldn't there be an else statement to be executed when mute == 0?

Reply:
No, if mute is 0, then the sound won't play. we don't need to do anything if it's muted